modules/setting/git_test.go

```diff
@@ -34,7 +32,34 @@ diff.algorithm = other
 `)
 	assert.NoError(t, err)
 	loadGitFrom(cfg)
-
-	assert.Len(t, GitConfig.Options, 1)
 	assert.EqualValues(t, "other", GitConfig.Options["diff.algorithm"])
 }
+
+func TestGitReflog(t *testing.T) {
+	oldGit := Git
+	oldGitConfig := GitConfig
+	defer func() {
+		Git = oldGit
+		GitConfig = oldGitConfig
+	}()
+
+	// default legacy reflog config
+	cfg, err := NewConfigProviderFromData(``)
+	assert.NoError(t, err)
+	loadGitFrom(cfg)
+
+	assert.EqualValues(t, "true", GitConfig.GetOption("core.logAllRefUpdates"))
+	assert.EqualValues(t, "90", GitConfig.GetOption("gc.reflogExpire"))
+
+	// custom reflog config
```

Comment:
I am confused by these comments.
Aren't you are basically testing the opposite thing of what the comments are saying?

Reply:
```suggestion
	// default reflog config without legacy options
	cfg, err := NewConfigProviderFromData(``)
	assert.NoError(t, err)
	loadGitFrom(cfg)

	assert.EqualValues(t, "true", GitConfig.GetOption("core.logAllRefUpdates"))
	assert.EqualValues(t, "90", GitConfig.GetOption("gc.reflogExpire"))

	// custom reflog config by legacy options
```

Reply:
Does 0b0984735485c1b5c2b95e0083e791285cf12925 look good to you?